packages/server/src/fhir/operations/expand.ts

```diff
@@ -266,32 +277,70 @@ async function computeExpansion(
     } else {
       await includeInExpansion(include, expansion, codeSystem, params);
     }
-
-    if (expansion.length > (count ?? MAX_EXPANSION_SIZE)) {
-      // Return partial expansion
-      break;
-    }
   }
 
+  await addExpandJob(valueSet, codeSystemCache); // Precompute expansion in the background for faster future searches
```

Comment:
Will this create a BullMQ job for every expansion, even if has been expanded before?

Reply:
This should only run after we needed to generate the expansion: once there are precomputed expansion rows in the DB, it won't queue the expansion job anymore